### PR TITLE
refactor/strict code

### PR DESCRIPTION
**Pull request by edwinmp**

### CODE REVIEW

File: src/routes/index.ts [hunk 1]
```diff
@@ -1,147 +1,119 @@
...
-export class Index {
+type ResponseFormat = 'json' | 'csv' | 'xml';
+export class Routes {
+  dbHandler: DB = new DB();
 
-    public dbConn: DB = new DB();
+  init(app: Application): void {
+    app.route('/')
+      .get((_req: Request, res: Response) => {
+        res.status(200).sendFile(join(__dirname + '/index.html'));
```
Comment (akmiller01) [inline src/routes/index.ts:13]: @edwinmp Does this line work for you? When I deploy via docker locally I get `Error: ENOENT: no such file or directory, stat '/src/dist/routes/index.html'`

**Review by akmiller01 — CHANGES_REQUESTED** [review]
Hey @edwinmp thanks for the refactor! I think I've just spotted one problem when deploying.

File: src/routes/index.ts (hunk 1 above)
Comment (edwinmp) [reply src/routes/index.ts:13]: hmm... was fine locally. Let me look into it.

Comment (akmiller01) [reply src/routes/index.ts:13]: I am using `docker-compose up` to deploy. Maybe I need to edit the `docker-compose.yml` to match something you're doing with docker?

Comment (edwinmp) [reply src/routes/index.ts:13]: This is fixed now. Merged in your fixes from the other branch as well, so I'm going to merge this into master.

**Pull request merged**